pyro/nn/auto_reg_nn.py

```diff
@@ -49,78 +119,67 @@ class AutoRegressiveNN(nn.Module):
         for any i, j in range(0, output_dim_multiplier) the subset of outputs [i, :] has identical
         autoregressive structure to [j, :]. defaults to `1`
     :type output_dim_multiplier: int
-    :param mask_encoding: a torch Tensor that controls the autoregressive structure (see reference). by default
-        this is chosen at random.
-    :type mask_encoding: torch.LongTensor
     :param permutation: an optional permutation that is applied to the inputs and controls the order of the
         autoregressive factorization. in particular for the identity permutation the autoregressive structure
-        is such that the Jacobian is upper triangular. by default this is chosen at random.
+        is such that the Jacobian is upper triangular. By default this is the identity permutation.
```

Comment:
"By default this is the identity permutation." 

isn't this incorrect?

Reply:
That's correct, let me change it

Reply:
I've made a subtle change in how the MADE masks are created so that it matches the method in TensorFlow rather than the exact description in the MADE paper.